crates/daphne/src/taskprov.rs

```diff
@@ -206,9 +239,13 @@ impl VdafConfig {
                     num_proofs,
                 },
             ) => {
-                const MAX_PROOFS: u8 = 3;
-                if num_proofs == 0 || num_proofs > MAX_PROOFS {
-                    return Err(DapAbort::InvalidTask { detail: format!("invalid number of proofs for Prio3SumVecField64MultiproofHmacSha256Aes128: got {num_proofs}; expected a value in range [0, {MAX_PROOFS}]"), task_id: *task_id });
+                if !(1..PRIO3_MAX_PROOFS + 1).contains(&num_proofs) {
```

Comment:
Reviewer note: previously we accepted 0 proofs here, but this doesn't make much sense.

Reply:
you can use `1..=PRIO3_MAX_PROOFS` instead of `+1` to have an inclusive range

Reply:
Done.